blog/2024-07-23-july-releases.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: "July 2024 Releases"
+slug: "2024-07-23-july-releases"
+authors: [beeankha]
+tags: [announcement, conda, conda-build, conda-libmamba-solver]
+description: |
+  Conda 24.7.0/.1, conda-build 24.7.0, and conda-libmamba-solver 24.7.0 have been released! 🎉
+image: img/blog/2024-07-23-july-releases/banner.png
+---
+
+The July 2024 releases included updates to conda, conda-build, and conda-libmamba-solver! 🎉 All of these have been released to both `main` and `conda-forge` channels.
+
+## Changes in Conda [24.7.0](https://github.com/conda/conda/releases/tag/24.7.0)
+
+To update conda, run:
+
+```bash
+conda install -n base conda=24.7.0
+```
+
+<!-- truncate -->
+
+### ✨ What's New? ✨
+
+- Add a new `reporters` setting for configuring output.
+- Report traceback of plugin loading errors with verbosity 2 or higher (`-vv` or more).
+- Skip checking for `.pyc` and `.pyo` files in the `conda doctor` "missing files" health check.
+- **Breaking change** `conda list --explicit` will not print authentication details by default. A new flag `--auth` has been added so folks can opt-in to this behavior.
+- Print transaction report for `@EXPLICIT` lockfile installs too.
+- Do not require `-n/--name` or `-p/--prefix` if `conda create` is invoked with `--dry-run`.
+- Add an `envvars_force_uppercase` setting which defaults to `True`, uppercasing all environment variables (thereby justifying `conda`'s current behaviour); when `envvars_force_uppercase` is set to `False`, conda will only save preserved-case variable names.
+- Alias `conda env list` command to `conda info --envs`.
+- Replace calls to `logger.warn` with `logger.warning`.
+
+### 🔧 What Got Fixed? 🔧
+
+- Improve treatment of logger levels.
+- Mask authentication details in `conda-meta/*.json` metadata.
+- Mask Anaconda.org tokens in verbose logs.
+- Fix parsing error when history file only contains a single commented line.
+- Add missing `emscripten` and `wasi` entries to the recognized platforms, and `wasm32` to the recognized architectures.
+- Fix checksum comparisons in `conda.gateways.connection.download.download()` to be case insensitive.
+- Disallow some more characters in Windows for prefix names (`^`, `%`, `!`, `=`, `(`, `)`, `\`). These characters complicate or prevent environment activation if present.
```

Comment:
We will most likely want to delete this line depending on what happens with https://github.com/conda/conda/pull/14065